src/sentry/conf/server.py

```diff
@@ -588,6 +588,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "sentry.tasks.update_user_reports",
     "sentry.tasks.user_report",
     "sentry.profiles.task",
+    "sentry.release_health.duplex",
```

Comment:
I am not sure what the implications of this is in production tbh

Reply:
Looking at the `sentry.releasehealth.metrics.compare` statsd metric, I would also say 200/minute.